word_frequency.py

```diff
@@ -1,19 +1,51 @@
+import re
+
 STOP_WORDS = [
     'a', 'an', 'and', 'are', 'as', 'at', 'be', 'by', 'for', 'from', 'has', 'he',
     'i', 'in', 'is', 'it', 'its', 'of', 'on', 'that', 'the', 'to', 'were',
     'will', 'with'
 ]
 
+def flatten_lol(lol):
+  flat_list = []
+  for l in lol:
+    for word in l:
+      flat_list.append(word)
+  return flat_list
 
 def print_word_freq(file):
-    """Read in `file` and print out the frequency of words in that file."""
-    # Your code will go here. You can write additional functions if you want, and call them inside this function.
-    # first open the file
+   #empty lists 
+    first = []
+    frequency = {}
+    
+
     with open(file) as f:
-        lyrics = f.readlines()
-        word_list = [line.split() for line in lyrics]
-        print(word_list)
+       #grab the item/words out of the text file
+      for items in f:
+          first.append(items)
+        #remove the punctuation from the text  
+      cleaned_text = []
+      for w in first:
+        clean = re.sub(r"[!?.,]","", w.lower())
+        cleaner = clean.split()
+        cleaned_text.append(cleaner)
+        
+    #run the clean text through the flatten function
+      working_list = flatten_lol(cleaned_text)
+      
 
+      for word in list(working_list):  
+          if word in STOP_WORDS:
+           working_list.remove(word)
+           for word in working_list:
+             count = frequency.get(word,0)
+             frequency[word] = count + 1
+     
+             frequency_list = frequency.keys()
+            #counting the words in the text
+             for words in frequency_list:
+               
+              print(words, '|' ,frequency[words]) 
```

Comment:
Great job on this Antoine! I like that you're commenting on the code as you go. It's going to really help in the transition when applying for roles.